### PR TITLE
update sbt-catalyst to 0.3.1 with cats 1.0.0-RC1

### PR DESCRIPTION
**Pull request by wedens**
Can you please publish a version after (if) those changes are merged?

### CODE REVIEW

**Comment by kailuowang** [comment]
Thanks so much. Tut in sbt-catalyst 0.3 is still broken with scalameta , will you wait for 0.4 which I will push tomorrow?

**Comment by wedens** [comment]
Yeah, sure.

**Comment by kailuowang** [comment]
sbt-catalysts 0.3.1 released, that should fix the tut error, do you want to give it a try?

File: build.sbt [hunk 1]
```diff
@@ -114,7 +113,7 @@ lazy val docs = project
       "white-color"       -> "#FFFFFF"),
     ghpagesNoJekyll := false,
     micrositeAuthor := "Kailuo Wang",
-    tutScalacOptions ~= (_.filterNot(Set("-Ywarn-unused-import", "-Ywarn-dead-code"))),
+    scalacOptions in Tut ~= (_.filterNot(Set("-Ywarn-unused-import", "-Ywarn-dead-code"))),
```
Comment (kailuowang) [inline build.sbt:116]: to get `Tut` working properly with scalameta, also need to get rid off the `% Provide` in line 145

**Comment by wedens** [comment]
Yeah, it works after update to 0.3.1.

**Comment by kailuowang** [comment]
Thanks so much!

**Comment by kailuowang** [comment]
Will release soon

**Pull request merged**